lib/src/score_emitter.dart

```diff
@@ -1,14 +1,14 @@
-part of benchmark_harness;
-
 abstract class ScoreEmitter {
+  const ScoreEmitter();
```

Comment:
Is this required in order to have the const constructor for ScoreEmitter?  If not, and the analyzer doesn't require it, I wouldn't add it.